src/pudl/transform/eia860.py

```diff
@@ -379,6 +407,27 @@ def utilities(eia860_dfs, eia860_transformed_dfs):
         'Y': 'NY',  # Typo
     })
 
+    # Combine phone number columns into one
+    def _make_phone_number(col1, col2, col3):
+        """Combine columns to make full phone number seperated by dashes."""
+        return (
+            col1.astype('string')
+            + '-' + col2.astype('string')
+            + '-' + col3.astype('string')
+        )
```

Comment:
If these columns are being stored as numbers in the spreadsheet, we might want to verify that what comes out of this matches a regular expression describing US phone numbers, since leading zeros would get dropped, and I think the 4-digit part of the phone number can have leading zeroes.

Reply:
hmmm that's a great point! I'll check it out